src/peoplemeasurement/csv_imports/sensor_csv_importer.py

```diff
@@ -9,22 +9,9 @@
 
 
 class SensorCsvImporter(CsvImporter):
+    model = Sensors
 
-    def _import_csv_reader(self, csv_reader) -> int:
-        with transaction.atomic():
-            if Sensors.objects.count() > 0:
-                self._truncate_sensors()
-
-            sensors = []
-            for row in csv_reader:
-                sensors.append(self._create_sensor_for_row(row))
-
-            if sensors:
-                Sensors.objects.bulk_create(sensors)
-
-        return len(sensors)
-
-    def _create_sensor_for_row(self, row):
+    def create_obj_dict_for_row(self, row):
```

Comment:
misschien hernoemen naar `create_model_instance_for_row(self, row)`, want we doen niet echt iets met een dict.